app/styles/app/modules/request/preview.scss

```diff
@@ -0,0 +1,27 @@
+.request-configs-preview-container {
+  display: flex;
+
+  & > div {
+    flex-grow: 1;
+    align-items: stretch;
+    width: 50%;
+  }
+
+  .left {
+    margin-right: 0.5em;
+  }
+
+  .right {
+    margin-left: 0.5em;
+  }
+
+  pre,
+  div > div {
+    height: calc(100% - 45px);
+    border: 1px solid $dry-cement;
+  }
+
+  div > div {
+    padding: 10px 20px;
+  }
```

Comment:
I'd like to suggest changing these selectors to use classes instead of tags, to make styles less dependent on the template markup structure, and thus, less fragile.

Reply:
makes sense. Sophie had a similar request on the previous branch. 

done: https://github.com/travis-ci/travis-web/commit/ee375ae73174a2030c5bd731ea2165ddb222e474